src/components/common/ShareAppButton.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { ElementType } from 'react';
+import { Alert, Share, StyleSheet, View } from 'react-native';
+import { Icon, TouchableOpacity } from '.';
+import config from '../../config/config';
+
+interface Props {
+  locale: 'he'|'en'|'ar'|'am'|'ru',
+}
+
+const ShareAppButton: ElementType = ({ locale }:Props) => {
+  const onShare = async () => {
+    try {
+      const downloadLink = config.downloadMohLandingPage[locale];
+
+      await Share.share({
+        message: `Hey, Please download. ${downloadLink}`,
```

Comment:
What about translating the message?

Reply:
please see my conversation with @ItayElgazar